src/composables/Project.js

```diff
@@ -296,6 +295,32 @@ export async function createProjectFolder(projectId, path) {
   });
 }
 
+/**
+ * Create a new directory. Ignore already exists error and its parents if necessary/not existing.
```

Comment:
Create a new directory and its parents if not existing. Ignore already exists error.